pkg/estimator/pb/service.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package github.com.karmada_io.karmada.pkg.estimator.pb;
+
+import "github.com/karmada-io/karmada/pkg/estimator/pb/generated.proto";
+
+// Package-wide variables from generator "generated".
+option go_package = "pb";
+
+service Estimator {
+  rpc CalClusterAvailableReplicas(CalAvailableReplicasRequest) returns (CalAvailableReplicasResponse) {}
```

Comment:
@kevin-wangzefeng Do you have any recommendations about the method's name? 
I propose using the `MaxReplicaPrediction`.

Reply:
how about `QueryMaxAvailableReplicas()` or just `MaxAvailableReplicas()`

Reply:
I vote for `MaxAvailableReplicas`.

Reply:
> MaxAvailableReplicas()

+1

Reply:
Parameter: `CalAvailableReplicasRequest` --> `MaxAvailableReplicasRequest`?
Returns: `CalAvailableReplicasResponse` --> `MaxAvailableReplicasRequestResponse`?

Reply:
Returns: `CalAvailableReplicasResponse` --> `MaxAvailableReplicasResponse`